compiler/main.bal

```diff
@@ -12,10 +12,15 @@ public type Options record {|
     boolean testJsonTypes = false;
     boolean showTypes = false;
     string? outDir = ();
+    string? gc = ();
 |};
 
 const SOURCE_EXTENSION = ".bal";
 public function main(string[] filenames, *Options opts) returns error? {
+    if !isValidGCStrategy(opts.gc) {
+        return error("Invalid GC strategy " + <string>opts.gc);
```

Comment:
not a good idea to cast, try creating a type for `gc` instead using string.

Reply:
Changed.